packages/library-sync/src/commonMain/kotlin/io/realm/mongodb/internal/KtorNetworkTransport.kt

```diff
@@ -45,6 +45,7 @@ import kotlinx.coroutines.CancellationException
 import kotlinx.coroutines.CoroutineDispatcher
 import kotlin.collections.set
 
+// TODO Consider adding a logger
```

Comment:
We definitely need logging, but there are issues with the setup right now as it is tied to Realm lifecycles in order to be able to control the debug level.

Maybe we need to add a System logger that users have very little control over, but that is for another PR no matter what.

Reply:
I guess we should just add an option to put in a logger in the `AppConfiguration` and that should be propagated to as the default for the `SyncConfiguration`? But I guess it fits best in on of the AppConfiguration PRs